package.json

```diff
@@ -1,23 +1,6 @@
 {
-  "name": "eslint-config-mavenlint",
-  "version": "2.2.2",
-  "description": "Mavenlink ESLint config",
-  "main": "index.js",
-  "repository": "https://github.com/mavenlink/mavenlint.git",
-  "license": "MIT",
-  "devDependencies": {
-    "eslint": "^4.14.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-react": "^7.5.1"
-  },
-  "peerDependencies": {
-    "eslint": ">=4.14.0",
-    "eslint-plugin-import": ">=2.8.0",
-    "eslint-plugin-jsx-a11y": ">= 5.1.1",
-    "eslint-plugin-react": ">=7.5.1"
-  },
-  "dependencies": {
-    "eslint-config-airbnb": "15.1.0"
-  }
+  "private": true,
```

Comment:
Why private?

Reply:
The root-level package.json is not what we want to publish. Instead, each package.json under the packages/ directory gets published.

From what I can see, this one usually gets stuff needed to test all of the individual packages together, and or scripts and whatnot.